test/python/test_create.py

```diff
@@ -26,3 +28,19 @@ def test_project(default_project):
 
     process = subprocess.run(['.rocks/bin/luatest'], cwd=project.path)
     assert process.returncode == 0, "luatest failed"
+
+
+def test_project_recreation(default_project):
```

Comment:
May be it's better to move this test into separate PR and push it firstly?

I suppose that "refactoring" shouldn't change behaviour

Reply:
Ok, I will do it.